packages/react-components/react-migration-v0-v9/src/stories/FormFieldShim/index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import descriptionMd from './Description.md';
+
+import { FormField } from '@fluentui/react-northstar';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { InputField } from '@fluentui/react-components/unstable';
+import { FormFieldShim } from '../../components/FormField/FormFieldShim';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    gridTemplateRows: '1fr',
+    width: 'fit-content',
+    alignContent: 'center',
+    alignItems: 'center',
+    justifyItems: 'center',
+    columnGap: '10px',
+    rowGap: '10px',
+  },
+  componentName: {
+    justifySelf: 'end',
+    ...shorthands.margin(0, '10px', 0, 0),
+  },
+});
+
+export const Default = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <h3>v0</h3>
+      <h3>shim</h3>
+      <h3>v9</h3>
+      <FormField required>
+        <input />
+      </FormField>
+      <FormFieldShim required>
```

Comment:
it looks different than v0 FormField - is it expected? 
<img width="360" alt="image" src="https://user-images.githubusercontent.com/13742664/212378766-a2856a5f-5168-4486-b5a8-99864fee355f.png">

Reply:
So, in this example V0 and the Shim components are looking the same because both have `<input />`  as their control. I can change that and use V0 or V9 `Input` in the shim.